### ISSUE TITLE
project name suggestion =)

### ISSUE THREAD

**Issue by mluu510** [M2kanderson/Cumulonimbus#1]
cool project! although the name is hard to pronounce and remember. maybe consider a project rename? i suggest 'Sound Nimbus'

**Comment by M2kanderson** [comment]
Good call.  I've also noticed it's difficult to type as a url.  We'll look into a rename.  Let us know if you have any other suggestions.
